### ISSUE TITLE
When visualized,I got outputs with black photos ,no results. Why? 

### ISSUE THREAD

**Issue by SpadesQ** [shekkizh/FCN.tensorflow#77]
After training.
When visualized,I got outputs with black photos ,no results.
Why?

**Comment by liuxymvp** [comment]
I met the same problem. So have you solved it ?

**Comment by cocacc** [comment]
I met the same problem. So have you solved it ? black photos , Why?

**Comment by WillSmisi** [comment]
I met the same problem. Have someone solved it ? black photos ,Thanks in advanced

**Comment by JoeDelK** [comment]
I me the same problem.... Someone solved? Thanks in advanced

**Comment by yimenhfeifei** [comment]
I don't know how to visualize pic in the code? Does it change tf.flags.DEFINE_string('mode', "visualize", "Mode train/ test/ visualize") like this? That is?  can anyone help me? thanks!!!

**Comment by Mingtzge** [comment]
@yimenhfeifei 
yes ,you're right!

**Comment by CLegendBuptSun** [comment]
Maybe I can explain why we got black outputs.
When I run the code on 1 GPU Titan Xp (memory 12GB), the original code's performance is good.(Batchsize=2 ,lr=1e-4,itr=30000)
When I run the code on 2 GPU Titan Xp (memory 24GB), the original code's performance is terrible.(Batchsize=2 ,lr=1e-4,itr=30000).But I changed the Batchsize=8 and lr=1e-4 when itr<20000,lr=1e-5 when lr>20000,the performance is better than run on 1GPU.
So your configuration of computer may affect your results.
So try to change these basic arguments, you will get the satisfied output.
